docs/modding-and-development/development/reviewing.md

```diff
@@ -54,7 +57,7 @@ In your final comment make sure to mention what you did, e.g. which aspects of t
 After leaving your comment, select the type of feedback.
 
 * **Comment**: General remarks, use if neither of the later two applies.
-* **Approve**: You found no issues when looking at the code or during testing.
+* **Approve**: You found no issues when looking at the code or during testing. **Only approve a PR if you'd be willing to maintain the code change in question. Merging a pull request is essentiall handing off responsibility of the changed code from the PR contributor to the codebase maintainer!**
```

Comment:
Spelling mistake >:(

Reply:
Fixed in 489e966d95bbac4929134624bd8d45f6123d24ad